docs/CONTRIBUTING.md

```diff
@@ -32,3 +32,75 @@ Coming soon!
 After values are pushed the cluster's drone will do a deployment run. Helmfile diff is used targeting `$CLOUD` & `$CLUSTER` before the final deployment of the artifacts.
 
 If you have any meaningful additions to this, please let us know!
+
+## Schema validation
+
+### Guidelines/best practices
+
+We intend to be very exact in our speech regarding schema validation. Currently we perform meta-schema validation on the `values-schema.yaml` file. By having accurate and consistent descriptions (using the `description` keyword), we can provide the user with a better experience in his configuration management of the `otomi-values` repository.
```

Comment:
First of all, even though I like your literate style of writing, I do think it is bloated and suggest you write more concise and to the point. Ask yourself questions like "What am I trying to convey?" and "How can I say this as simple and to the point as possible?". Redacting your own writing a couple of times usually works wonders. 

Second, I don't think adding instructions on how to create good schema files should go here. Maybe add a link to the instructions themselves. Again, we will NOT be copying instructions that come with the open source tools and solutions we use. 

So please find where that information lives and add a sentence guiding people there.

Reply:
Yes I agree, it's just my first draft. When I read it again I can usually rewrite it much better.

Reply:
> Second, I don't think adding instructions on how to create good schema files should go here. Maybe add a link to the instructions themselves. Again, we will NOT be copying instructions that come with the open source tools and solutions we use.

Yes that makes sense, will look at it now.